src/Core/Domain/Order/Command/AddOrderFromBackOfficeCommand.php

```diff
@@ -139,7 +140,10 @@ private function assertIsModuleName($moduleName)
     private function assertOrderStateIsPositiveInt($orderStateId)
     {
         if (!is_int($orderStateId) || 0 >= $orderStateId) {
-            throw new OrderException('Invalid order state id');
+            throw new InvalidOrderStateException(
+                InvalidOrderStateException::NOT_FOUND,
```

Comment:
```suggestion
                InvalidOrderStateException::INVALID_ID,
```

`NOT_FOUND` would be more adapted for a valid ID that doesn't match any row in the database

I hesitate between `INVALID_ID` because it is redundant with the class name, I thought about `MISSING` at first 🤔

Reply:
Good idea for `MISSING` but empty is not missing. I prefer `INVALID_ID` ;)

Reply:
Fixed